add_package.rb

```diff
@@ -103,20 +126,19 @@ def fix_permissions(platforms)
 end
 
 def run_commands(commands)
-  hl = HighLine.new
-  hl.say "running commands..."
+  @hl.say "running commands..."
   @command_failures = []
   commands.each do |command|
-    hl.say(command)
+    @hl.say(command)
     cmd = Mixlib::ShellOut.new(command)
     cmd.run_command
     @command_failures << command unless cmd.status.exitstatus == 0
   end
 
   unless @command_failures.empty?
-    hl.say "<%= color('The following commands failed:', BOLD) %>"
+    @hl.say "<%= color('The following commands failed:', BOLD) %>"
     @command_failures.map {
-      |cmd| log_string = hl.color(cmd, :red) ; hl.say(log_string)
+      |cmd| log_string = @hl.color(cmd, :red) ; @hl.say(log_string)
```

Comment:
Now that `purge_cdn` is being added after `run_commands`, perhaps we should add another instance of `@hl.agree` to confirm that we want to continue with failed commands?

Note that currently the mkdir commands are always reported to fail when the directories already exist...